tests/foreman/installer/test_infoblox.py

```diff
@@ -15,183 +17,162 @@
-
-    :id: 75650c06-79b6-11e6-ad91-68f72889dc7f
-
-    :Steps: Enable Infoblox plugin via installer option
-        --enable-foreman-proxy-plugin-dhcp-infoblox
-
-    :expectedresults: Check DHCP plugin is enabled on host
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-def test_disable_dhcp_plugin():
-    """Check Infoblox DHCP plugin can be disabled on host
-
-    :id: ea347f34-79b7-11e6-bb03-68f72889dc7f
-
-    :Steps: Disable Infoblox plugin via installer
-
-    :expectedresults: Check DHCP plugin is disabled on host
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-@pytest.mark.upgrade
-def test_dhcp_ip_range():
-    """Check host get IP from Infoblox IP range while provisioning a host
-
-    :id: ba957e82-79bb-11e6-94c5-68f72889dc7f
-
-    :Steps: Provision a host with infoblox as dhcp provider
-
-    :expectedresults: Check host ip is on infoblox range configured by
-        option --foreman-proxy-plugin-dhcp-infoblox-use-ranges=true
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
-    """
-
-
-@pytest.mark.stubbed
-@pytest.mark.tier3
-@pytest.mark.upgrade
-def test_dns_records():
-    """Check DNS records are updated via infoblox DNS plugin
-
-    :id: 007ad06e-79bc-11e6-885f-68f72889dc7f
-
-    :Steps:
-
-        1. Provision a host with infoblox as dns provider
-        2. Update a DNS record on infoblox
-
-    :expectedresults: Check host dns is updated accordingly to infoblox
-
-    :CaseLevel: System
-
-    :CaseAutomation: NotAutomated
+import requests
+from fauxfactory import gen_mac
+from fauxfactory import gen_string
+from requests.exceptions import HTTPError
+
+from robottelo.config import settings
+from robottelo.utils.installer import InstallerCommand
+
+
+infoblox_plugin_enable = [
+    'enable-foreman-proxy-plugin-dhcp-infoblox',
+    'enable-foreman-proxy-plugin-dns-infoblox',
+]
+infoblox_plugin_opts = {
+    'foreman-proxy-dhcp': 'true',
+    'foreman-proxy-dhcp-managed': 'false',
+    'foreman-proxy-dhcp-provider': 'infoblox',
+    'foreman-proxy-dhcp-server': settings.infoblox.hostname,
+    'foreman-proxy-plugin-dhcp-infoblox-dns-view': 'default',
+    'foreman-proxy-plugin-dhcp-infoblox-network-view': 'default',
+    'foreman-proxy-plugin-dhcp-infoblox-username': settings.infoblox.username,
+    'foreman-proxy-plugin-dhcp-infoblox-password': settings.infoblox.password,
+    'foreman-proxy-plugin-dhcp-infoblox-record-type': 'fixedaddress',
+    'foreman-proxy-dns': 'true',
+    'foreman-proxy-dns-provider': 'infoblox',
+    'foreman-proxy-plugin-dns-infoblox-username': settings.infoblox.username,
+    'foreman-proxy-plugin-dns-infoblox-password': settings.infoblox.password,
+    'foreman-proxy-plugin-dns-infoblox-dns-server': settings.infoblox.hostname,
+    'foreman-proxy-plugin-dns-infoblox-dns-view': 'default',
+}
+
+
+@pytest.mark.stream
```

Comment:
can we remove this mark as this test is not entirely for stream and handle the part of the close loop for a stream with `sat.version` ?

Reply:
It is for stream, but yes it will work as the codebase for the BZ is present in 6.13 as well

Reply:
Cool, we can cherrypick this to 6.13 and older versions as well, updating the labels ..